src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -130,6 +131,8 @@ public static class ScriptConstants
         public const string ExtensionBundleTemplatesFile = "templates.json";
         public const string ExtensionBundleResourcesFile = "Resources.json";
 
+        public const string HstsIsEnabledPropertyName = "isEnabled";
```

Comment:
I don't think you need this. The config binding should be able to handle it since the name matches exactly.